app/src/lib/helpers/regex.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * get all regex catpures within a body of text
```

Comment:
This is extremely minor, but since this pull request is still open I figured it was worth mentioning this typo. 'catpures'=>'captures'

Added as a comment.

Reply:
🦅 👀